specs/IR-0001/suite/test_spec_ir0001.py

```
@@ -0,0 +1,357 @@
+        """INV/BEH/IFACE 义务动词必须是强规范动词（必须/不得/禁止）——"应/须"可被原则上/建议稀释。"""
+        s = read(SPEC)
+        clauses = re.findall(r"^- ((?:INV|BEH|IFACE)-\d+): (.+)$", s, re.M)
+        self.assertGreaterEqual(len(clauses), 14, "INV/BEH/IFACE 条款不足 14 条")
+        for uid, text in clauses:
+            self.assertTrue(any(w in text for w in self.STRONG_MODALITY),
+                            f"{uid} 缺强规范动词（必须/不得/禁止）——义务降级改写嫌疑")
+
+    def test_no_weakening_phrases(self):
+        """条款行与 AC 段禁弱化逃逸短语——把强制条款掏空为意向声明的直接红旗。"""
+        s = read(SPEC)
+        lines = [l for l in s.splitlines()
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)